tests/harness/verify_proofs.py

```
@@ -0,0 +1,53 @@
+  - Deterministic replay digest recomputation and comparison
+  - Invariant checks (append‑only, FF‑only, temporal monotonicity, etc.)
+"""
+
+from __future__ import annotations
+import argparse
+import os
+import sys
+
+
+def parse_args(argv: list[str]) -> argparse.Namespace:
+    p = argparse.ArgumentParser(description="Ledger‑Kernel proof verifier (skeleton)")
```

Ruff (0.14.2)

String contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)